static/js/portico/header.js

```diff
@@ -6,41 +6,39 @@ $(() => {
         return false;
     });
 
+    const isMobileWidthHeader = window.matchMedia("(hover: none)").matches;
```

Comment:
Let's call this `is_touchscreen`, since that's what we mean -- it has nothing to do with width now.

We also use underscores for variable names in this codebase.

Reply:
My bad, my brain kind assumed JS was always camelcase. Should be fixed now.